### PR TITLE
fix: markup shown as tags from some browsers

### PR DESCRIPTION
**Pull request by rubiin**
Chromium notification shows raw HTML. This fixes that

context https://github.com/Aylur/ags/issues/480

Before:

image

After:
image

### CODE REVIEW

**Comment by rubiin** [comment]
image

When the notification body is long, the limit  set on label via `max_width_chars` doesnt work,

The latest commit fixes this as well


After the commit

image

File: options.ts [hunk 1]
```diff
@@ -1222,6 +1222,11 @@ const options = mkOptions(OPTIONS, {
     },
 
     dummy: opt(true),
+    hyprland: {
+        gaps: opt(2.4),
+        inactiveBorder: opt('#282828'),
+        gapsWhenOnly: opt(false),
+    },
```
Comment (Jas-SinghFSU) [inline options.ts:1229]: Was this added by mistake?

**Comment by Jas-SinghFSU** [comment]
LGTM otherwise

File: options.ts (hunk 1 above)
Comment (rubiin) [reply options.ts:1229]: oh yeah that was a mistake. Should have been on another branch

**Comment by Jas-SinghFSU** [comment]
Sweeet, let's merge it!

**Comment by rubiin** [comment]
> image
> 
> When the notification body is long, the limit set on label via `max_width_chars` doesnt work,
> 
> Use this command to test
> 
> `notify-send -u critical "Build failed!" "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2M2UzNDVlYTAwNmEwMjZhNjVkMzc0MzUiLCJyb2xlcyI6WyJTVVBFUkFETUlOIl0sImlhdCI6MTczMDcxODA5NywiZXhwIjoxNzYyMjU0MDk3LCJhdWQiOiJuZXN0aWZ5IiwiaXNzIjoibmVzdGlmeSIsInN1YiI6IjYzZTM0NWVhMDA2YTAyNmE2NWQzNzQzNSJ9.eY5c9AsXb0iFbjVtqI8jk1VMPSomjO3cHEa1HzZo1yY"`

I will have a separate PR for this bug I encountered. Its kind of an edge case

**Pull request merged**